minetest/minetest
# Fix get_craft_result when called before initHashes

Fixes #8607

`getCraftResult` was not working before the `initHashes` call because it uses the recipe priority which was not initialized until the hashes are inited.

## To do

This PR is Ready for Review.

## How to test

- Start a world in the minimal game, there should be no assertion failure
- In MTG, try placing a wooden stair or slab in the fuel slot of a furnace
